src/index.tsx

```diff
@@ -1,16 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
-import Header from './components/Header';
-import Footer from './components/Footer';
 
 ReactDOM.render(
-  <React.StrictMode>
-    <Header />
+  <BrowserRouter>
     <App />
```

Comment:
I might recommend moving all this to the `<App />` then have that component control all the flow and the router. It can be easier easier if you have all that centralized in one place.

Reply:
moved the router to the App component and returned the React.StrictMode to the index file